server/models/SentenceAndOffenceViewModel.ts

```diff
@@ -72,4 +76,12 @@ export default class SentenceAndOffenceViewModel {
   public isErsedChecked(): boolean {
     return this.userInputs?.calculateErsed === true
   }
+
+  public isErsedElligible(): boolean {
+    return this.sentencesAndOffences.filter(sentence => SentenceTypes.isSentenceErsedElligible(sentence)).length > 0
```

Comment:
Typescript has a `find` function instead of `filter`. Which means it will stop looping once it finds a sentence matching the condition.

Reply:
Done.